forge-gui/res/editions/Edge of Eternities.txt

```diff
@@ -6,30 +6,155 @@ Type=Expansion
 ScryfallCode=EOE
 
 [cards]
+# 1 R Anticausal Vestige @Chase Stone
 2 M Tezzeret, Cruel Captain @Chris Rahn
+# 4 R Astelli Reclaimer @Carly Milligan
+6 C Banishing Light @Rovina Cai
+14 U Emergency Eject @Leon Tukker
+# 15 M Exalted Sunborn @Scott M. Fischer
+# 19 R Haliya, Guided by Light @Kieran Yanner
 35 R The Seriema @Sergey Glushakov
+# 37 U Starfield Shepherd @Marta Nael
```

Comment:
You shouldn't need to comment out spoiled cards that aren't scripted yet.

Reply:
I'll know for next time. :)